apps/omg_eth/lib/omg_eth/release_tasks/set_contract.ex

```diff
@@ -84,24 +84,30 @@ defmodule OMG.Eth.ReleaseTasks.SetContract do
   end
 
   defp apply_static_settings(network) do
-    network =
+    _network =
```

Comment:
since now you don't need the network var, you can put this whole bit (`case`) in a function like `verify_network_name` (if we want such safeguard to be in place ofc). It can return `:ok` on success and do this `exit(@error)` on failure, to not complicate/change things too much